Project.toml

```
@@ -1,6 +1,6 @@
 name = "Unitful"
 uuid = "1986cc42-f94f-5a68-af5c-568840ba703d"
-version = "1.4.1"
+version = "1.4.2"
```

Thank you also for having increasing the version number!  However I feel like adding new functions falls into "adding new feature", so I'd bump to
```suggestion
version = "1.5.0"
```

---

OK, I changed it! Didn't want to be presumptuous about the size of the update ;)